iso4217.go

```diff
@@ -15,11 +15,11 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// Generated on 2023-06-02T09:32:24Z by adam, any modifications will be overwritten
+// Generated on 2023-06-05T03:51:27Z by raphaelsantodomingo, any modifications will be overwritten
 package iso4217
 
 type CurrencyCode struct {
-	Code, Name string
+	Code, Name, MicroUnit string
```

Comment:
Should `MicroUnit` be an int?

Reply:
yea I was debating whether to perform the conversion here or let users perform the conversion on it. Preference?

Reply:
I would render the code as an `int` so callers don't have to convert. We can save them a lot of time.

Reply:
I'd probably call it `DecimalPlaces int` too.

Reply:
sounds good, updating..

Reply:
done. Using `-1` to represent MicroUnits with value "-"

Reply:
Okay, we should mention that in the Go docs of the package.